pkg/image/apis/image/helper.go

```diff
@@ -2,21 +2,23 @@ package image
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"net"
 	"net/url"
 	"regexp"
 	"sort"
 	"strings"
 
-	"k8s.io/apimachinery/pkg/api/errors"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 
 	"github.com/blang/semver"
 	"github.com/golang/glog"
 
 	"github.com/openshift/origin/pkg/image/apis/image/internal/digest"
+	quotautil "github.com/openshift/origin/pkg/quota/util"
```

Comment:
This shouldn't be allowed.  Did the verify-imports slip?

Reply:
no it catched it, I will copy it into image import as that is the only place that uses it and make a private copy in legacy client as that is going away anyway